src/dotnet/commands/dotnet-compile/NativeCompiler.cs

```diff
@@ -22,16 +23,37 @@ public override bool Compile(ProjectContext context, CompilerCommandApp args)
 
             var managedOutput = outputPaths.CompilationFiles.Assembly;
 
+            // Create the library exporter
+            var exporter = context.CreateExporter(args.ConfigValue);
```

Comment:
AFAIK this still doesn't work unless I'm missing something fundamental about the CoreRT implementations.